RadarSDK/RadarSettings.m

```diff
@@ -240,8 +259,18 @@ + (RadarLogLevel)logLevel {
 }
 
 + (void)setLogLevel:(RadarLogLevel)level {
-    NSInteger logLevelInteger = (int)level;
-    [[NSUserDefaults standardUserDefaults] setInteger:logLevelInteger forKey:kLogLevel];
+    NSMutableDictionary *sdkConfiguration = [[RadarSettings clientSdkConfiguration] mutableCopy];
```

Comment:
I do not think it is appropriate to sync the log level here. I believe that the responsibility of  `RadarSettings` should be a simple wrapper around NSUserdefault, like our own key value store. Such syncing logic probably belongs to `RadarLogger` instead. (granted this will result in more lines of code changed)

Reply:
how this works is we have a client sdk config (what the client wants), then a server sdk config (what the server wants). and setLogLevel sets what the client wants, then that gets sent to the server and server can override a client config and send the config that will be used back down. 

RadarLogger is not going to be syncing the config every single time it gets the log level. The logic here is to minimize sync config calls if the target log level is already set. 

I think its fine to keep this here as this is the settings managing logic.(RadarSettings updateSessionId also does some logic to flush replays, getLogLevel does some logic to determine log level based on if the user is a debug user.